pages/common/az-bicep.md

```diff
@@ -0,0 +1,33 @@
+# az bicep
+
+> Bicep CLI command group.
+> Part of `azure-cli`.
+> More information: <https://docs.microsoft.com/cli/azure/config>.
+
+- Build a Bicep file:
+
+`az bicep build --file {bicep_file}`
+
+- Attempt to decompile an ARM template file to a Bicep file:
+
+`az bicep decompile --file {json_template_file}`
+
+- Install Bicep CLI:
+
+`az bicep install`
```

Comment:
`az bicep install` can be moved to the top because it seems to be the beginning step of this application.

Reply:
Committed!